### PR TITLE
python3-paramiko: update to 3.2.0

### PR DESCRIPTION
**Pull request by carvalhudo**
#### Testing the changes
- I tested the changes in this PR: **NO**

### CODE REVIEW

File: srcpkgs/python3-paramiko/template [hunk 1]
```diff
@@ -11,5 +11,5 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="http://www.paramiko.org/"
```
Comment (paper42) [inline srcpkgs/python3-paramiko/template:12]: https

Comment (paper42) [inline srcpkgs/python3-paramiko/template:12]:
```suggestion
homepage="http://www.paramiko.org/"
changelog="https://www.paramiko.org/changelog.html"
```

File: srcpkgs/python3-paramiko/template [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'python3-paramiko'
 pkgname=python3-paramiko
-version=2.11.0
-revision=3
+version=3.0.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-cryptography python3-pyasn1 python3-bcrypt python3-pynacl
```
Comment (paper42) [inline srcpkgs/python3-paramiko/template:7]: remove six, it's not required anymore

**Review by paper42 — COMMENTED** [review]
this is a major version bump, so this should be properly tested with all of reverse dependencies, as far as I can tell from the PR description, not even this library itself was tested.

File: srcpkgs/python3-paramiko/template (hunk 2 above)
Comment (carvalhudo) [reply srcpkgs/python3-paramiko/template]: done

File: srcpkgs/python3-paramiko/template (hunk 1 above)
Comment (carvalhudo) [reply srcpkgs/python3-paramiko/template]: suggestion implemented

Comment (carvalhudo) [reply srcpkgs/python3-paramiko/template]: done

**Comment by carvalhudo** [comment]
> this is a major version bump, so this should be properly tested with all of reverse dependencies, as far as I can tell from the PR description, not even this library itself was tested.

Yes, you are correct. I noticed you did the last version update for this package (2.7.2 -> 2.11.0). Do you have some suggestion regarding the tests we can make to validate such new version?This is my first package upgrade and I only use paramiko indirectly (through ansible).

**Pull request closed without merging**